tests/unit/loadDojoConfig.ts

```diff
@@ -0,0 +1,18 @@
+import { describe, it } from 'intern!bdd';
+import * as assert from 'intern/chai!assert';
+import loadDojoConfig from '../../src/loadDojoConfig';
+
+describe('loadDojoConfig', () => {
+	it('should load the specified module if it exists', () => {
+		const config = loadDojoConfig('tests/support/mocks/dojo.config.js');
```

Comment:
and if that was the case cli-build wouldn't have to worry about this at all, it would just be received in the command.